src/Security/DefaultAdminService.php

```diff
@@ -43,10 +51,11 @@ public function __construct()
     /**
      * Set the default admin credentials
      *
-     * @param string $username
+     * @param string $username 
      * @param string $password
+     * @param string $email
      */
-    public static function setDefaultAdmin($username, $password)
+    public static function setDefaultAdmin($username, $password, $email = null) 
```

Comment:
as in my previous comment, none of these changes should be made to this function